src/optionsStyles/font.coffee

```diff
@@ -109,7 +109,7 @@ defineOptionsStyle 'font', createReactClass
   render: ->
     lc = @props.lc
 
-    {div, input, select, option, br, label, span, optgroup} = React.DOM
+    {div, input, select, option, br, label, span, optgroup} = DOM
```

Comment:
It's more common to see this changed from `React.DOM` to `ReactDOM` than simply `DOM`.

Reply:
See my other comment, but basically `react-dom` only contains methods for rendering to the DOM, it doesn't contain the DOM element factories themselves, so it would be misleading to call this `ReactDOM`. I'll change the name of the shim to make it clear that it is a shim for `react-dom-factories`.